app/views/mobileNumber/ConfirmRemoveMobileView.scala.html

```diff
@@ -32,10 +34,44 @@
 
 @mainTemplate(pageTitle = messages("confirmRemoveMobile.title"), backLinkContent = Some(backLink)) {
 
-  <h1 class="govuk-heading-l">@messages("confirmRemoveMobile.heading", mobile)</h1>
-
   @formWithCSRF(action = controllers.mobileNumber.routes.ConfirmRemoveMobileController.removeMobileNumber()) {
-    <input type="hidden" name="value" value="submit">
-    @govukButton(Button(content = Text(messages("base.confirmAndContinue"))))
+    @govukRadios(Radios(
```

Comment:
You need to add an error summary to this view for when there's an error in the form